### ISSUE TITLE
mouse move from code using XTEST extension does not work

### ISSUE THREAD

**Issue by thenik** [neutrinolabs/xorgxrdp#249]
Hello,

I am trying to do some automation for my xrdp session and it does not work.
**How to reproduce:**

```
sudo apt-get install xautomation
xte 'mousemove 200 200'
```

the same story from python code

```
import mouse
mouse.move(100, 100, absolute=True, duration=3)
```

When I run this code or xte from my notebook it works well as expected.

Any ideas how to manage this issue?

**Comment by matt335672** [comment]
@thenik - I can reproduce this on the Xorg backend with your clear instructions. We'll make it an issue against that.

If you use the Xvnc backend, you should find the above command works fine - it does for me.

**Comment by thenik** [comment]
**Here is my view no what did happen here:**
1) I connected from windows rdp client to linux \xrdp server.

2) when I ran mouse.move command on the server X session it moved mouse
but in 1 sec or even less windows rdp client moved mouse to original point because client has higher proirity.

So I see only 1 way how to solve that is to find a way how to turn off mouse position from a rdp client.

**Comment by jsorg71** [comment]
When the pointer position is moved from the server side, we should send a PDU to update the client, see
https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/3058381e-c856-4b26-a93c-d8f5514f8c3c
I don't think we have ever implemented this.
Usually, one wants to run the recording software on the client so I don't think this has come up.

**Comment by AdamWHY2K** [comment]
Damn, just spent hours tryna figure out why xdotool over RDP was being so weird.

@jsorg71 implementing this would require changes in this repo and the xrdp repo right?

**Comment by jsorg71** [comment]
That's right.  here are the pointer update defines from common/ms-rdpbcgr.h
```
/* Server Pointer Update PDU: messageType (2.2.9.1.1.4) */
#define TS_PTRMSGTYPE_SYSTEM           1
#define TS_PTRMSGTYPE_POSITION         3
#define TS_PTRMSGTYPE_COLOR            6
#define TS_PTRMSGTYPE_CACHED           7
#define TS_PTRMSGTYPE_POINTER          8
```

Since we implemented TS_PTRMSGTYPE_SYSTEM last year, the only one not done is TS_PTRMSGTYPE_POSITION.
I think this is why xdotool does not work right.

It an easy feature to add but it will touch libxrdp xrdp xup, xvnc, and xorgxrdp.

**Comment by jsorg71** [comment]
I create 2 PRs for this.  Works with xdotool and XWarpPointer test program.
https://github.com/neutrinolabs/xrdp/pull/3714
https://github.com/neutrinolabs/xorgxrdp/pull/405

**Comment by AdamWHY2K** [comment]
Can confirm latest devel version fixes this issue, tysm for the quick response
